client/me/purchases/cancel-purchase/index.jsx

```diff
@@ -180,7 +180,7 @@ class CancelPurchase extends React.Component {
 		}
 
 		if ( isSubscription( purchase ) ) {
-			heading = this.props.translate( 'Cancel Your %(purchaseName)s Subscription', {
+			heading = this.props.translate( 'Cancel your %(purchaseName)s Subscription', {
```

Comment:
Any reason for changing this?  I think it's OK to all be capitalized (and https://capitalizemytitle.com/ seems to agree :smiley: )